### PR TITLE
string & bool docs

### PR DESCRIPTION
**Pull request by corymickelson**
also added link in cmake-js docs, and fixed typo in node-gyp, I dont know why the diff is showing the entire cmake-js and node-gyp md files.

### CODE REVIEW

File: doc/boolean.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Boolean
+
+You are reading a draft of the next documentation and it's in continuos update so
```
Comment (SlurpTheo) [inline doc/boolean.md]: typo: continuos

File: doc/string.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# String
+
+You are reading a draft of the next documentation and it's in continuos update so
```
Comment (SlurpTheo) [inline doc/string.md]: typo: continuos

**Comment by mhdawson** [comment]
Looks like this needs a rebase as it shows conflicts and changes in 2 unrelated files (cmake-js.md and node-gyp.md)

**Comment by mhdawson** [comment]
@corymickelson

**Pull request closed without merging**